cmd/clusterctl/BUILD

```diff
@@ -17,6 +16,5 @@ go_binary(
     name = "clusterctl",
     embed = [":go_default_library"],
     visibility = ["//visibility:public"],
+    pure = "on",
```

Comment:
Is this a valid option for go_library? I can't find it in the docs https://github.com/bazelbuild/rules_go/blob/master/go/core.rst#go-library

Reply:
It is not supported for `go_library` afaict, but it is set on a `go_binary` here.